pkg/operation/hybridbotanist/addons.go

```diff
@@ -122,6 +122,11 @@ func (b *HybridBotanist) generateCoreAddonsChart() (*chartrenderer.RenderedChart
 	if err != nil {
 		return nil, err
 	}
+	vpnShoot, err = b.ShootCloudBotanist.UpdateVpnShootConfig(vpnShoot)
```

Comment:
Move that part up to line 92 and change:

```golang
vpnShootCloudSpecific, err := b.ShootCloudBotanist.GenerateVPNShootConfig()
if err != nil {
	return nil, err
}
vpnShoot = utils.MergeMaps(vpnShoot, vpnShootCloudSpecific)
```

(Similar to https://github.com/gardener/gardener/blob/master/pkg/operation/hybridbotanist/addons.go#L186-L189 and https://github.com/gardener/gardener/blob/master/pkg/operation/hybridbotanist/addons.go#L191-L197)

Also, run `make verify` before comitting.

Reply:
OK, we can do it that way.

Reply:
Done. I cannot get `make verify` to run correctly on my environment, but I cannot see the output of ci either...

Reply:
Got it running, `make verify` passed cleanly.